docs/running_psalm/error_levels.md

```diff
@@ -4,6 +4,9 @@ You can run Psalm in at different levels of strictness from 1 to 8.
 
 Level 1 is the most strict, level 8 is the most lenient.
 
+When no level is explicitly defined, psalm defaults to level 2.
```

Comment:
Are you sure about this? I thought psalm scans the codebase and determines a appropriate level.

Reply:
I guess its correct, see the cited code in the PR description.

Reply:
> Are you sure about this? I thought psalm scans the codebase and determines the appropriate level from that.

I guess there is some kind of `init` command which guesses a proper default level when you use it.
I am talking about what psalm does when no level is defined in the config file at all.